src/translations/en.json

```
@@ -1982,14 +1985,17 @@
                 "green": "Green",
                 "red": "Red",
                 "yellow": "Yellow"
-              }
+              },
+              "description": "Gauge card is a basic card that allows visually seeing sensor data."
             },
             "glance": {
               "name": "Glance",
-              "columns": "Columns"
+              "columns": "Columns",
+              "description": "Glance cards are very compact. Very useful to group together multiple sensors for a quick and easy overview."
```

Stays the same without that additional word.

"very compact" ? Is that clear? Maybe just compact or..

`Glance cards are useful to group multiple sensors in a compact overview.`

```suggestion
              "description": "Glance cards are very compact. Very useful to group multiple sensors for a quick and easy overview."
```

Finally, this is talking about plural cards, while the order sentences are singular (`card` vs `cards`), which is inconsistent.

---

Have you considered the alternative I listed?

---

I believe I changed it to the alternate you listed

---

`The Glance card is very compact. Very useful to group multiple sensors for a quick and easy overview.`

---

oh
I see

---

Changed